GVRf/Framework/framework/src/main/jni/objects/components/mesh_collider.h

```diff
@@ -40,7 +42,18 @@ class MeshCollider: public Collider {
     }
 
     Mesh* mesh() const {
-        return mesh_;
+        if (mesh_) {
```

Comment:
I would like to understand the reason for all these nullptr checks. Here and in render_data.cpp.